src/query.rs

```diff
@@ -691,3 +691,32 @@ impl ElementQueryable for WebDriver {
         ElementQuery::new(ElementQuerySource::Driver(&self.session), poller, by)
     }
 }
+
+//
+// Send test (It passes if it compiles)
+//
+
+async fn _test_is_send() -> WebDriverResult<()> {
```

Comment:
Consider removing the blank line to make it more obvious that the comment belongs with the function. Should it have /// instead?

Also please add an extra sentence to make it clearer that this code is not intended to be executed. Thanks